java-checks/src/test/files/checks/FieldModifier.java

```diff
@@ -1,4 +1,17 @@
 class A {
+  // intentionally don't have modifier
+  int b; // Compliant
+  // comment
+  int a; // Noncompliant
+
+  // modifier
+  // comment
+  int c; // Compliant
+
+  // comment2
+  // comment
+  int d; // Noncompliant
+
```

Comment:
1. We could add one more test case to show that things like `// FooBarmodifierxyz` are also compliant
2. If you chose to merge the above suggestion about ignoring case sensitivity, we also should add a testcase like `// Modifier for visibility intentionally ommitted.` (i.e., containing some uppercase chars).